R/rt_source_uses.R

```diff
@@ -2,31 +2,23 @@
 #'
 #' TaxRef references many publications across its documentation. The different
 #' sources are used for different reasons this function returns the list of
-#' usages for a source.
+#' usages for a source when no arguments are provided, otherwise returns the
+#' specific type of use.
 #'
-#' @export
-rt_source_uses = function() {
-
-  api_query = rt_GET("sourceUses")
-
-  parse_taxa(api_query)
-}
-
-#' Retrieve a source use in TaxRef
-#'
-#' @param usage_id {`numeric(1)`}\cr{}
-#'                 the id of a source use in TaxRef see
-#'                 [`rt_source_uses()`]
+#' @param usage_id {`numeric(1)`, default = `NULL`}\cr{}
+#'                 the id of a source use in TaxRef
 #'
 #' @examples
-#' rt_source_uses_id(usage_id = 11)
+#' rt_source_uses(276)
```

Comment:
I think it's better to use the arg name in examples:

```suggestion
#' rt_source_uses(usage_id = 276)
```

Reply:
Yes I've missed this one.